app/controllers/application_controller.rb

```diff
@@ -31,4 +32,8 @@ def layout_by_resource
       "application"
     end
   end
+
+  def available_roles
+    current_user.admin? ? User::ROLES.reject { |role| role == User::SUPER_ADMIN_ROLE } : User::ROLES
```

Comment:
suppose `User::SUPER_ADMIN_ROLE` shouldn't be visible from UI at all...
@igorkasyanchuk , correct?

Reply:
super admin could create other super admins